File: backend/components/metiers/administration_publique/utils/audit/samples/sample_audit_helper.test.py
# Test du helper d’audit d’exemple (Python)
from .sample_audit_helper import sample_audit_log
def test_sample_audit_log(capsys):
    assert sample_audit_log('user123', 'LOGIN') is True
    captured = capsys.readouterr()
    assert '[AUDIT] Utilisateur: user123, Action: LOGIN' in captured.out

File: backend/components/metiers/administration_publique/utils/audit/samples/sample_audit_helper.test.js
// Test du helper d’audit d’exemple (JS)
const { sampleAuditLog } = require('./sample_audit_helper');
describe('sampleAuditLog', () => {
  it('should log and return true', () => {
    expect(sampleAuditLog('user123', 'LOGIN')).toBe(true);
  });
});
